app/coffeescript/map.coffee

```diff
@@ -27,10 +28,6 @@ define [
           draggable: true
           panControl: false
       markers:
-        mapPin:        "/assets/nonsprite/map/map_pin_red4.png"
```

Comment:
If you set the values to either `undefined` or `null` (can't remember which), then FlightJS makes it required.